app/src/main/java/com/example/placefinder/MainActivity.kt

```diff
@@ -16,7 +16,7 @@ import com.example.placefinder.ui.theme.ParkFinderTheme
         override fun onCreate(savedInstanceState: Bundle?) {
             super.onCreate(savedInstanceState)
             setContent {
-                PlaceFinderTheme {
+                ParkFinderTheme {
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.